src/databricks/labs/ucx/install.py

```diff
@@ -480,11 +481,23 @@ def warehouse_type(_):
                 cluster_policy = json.loads(self._prompts.choice_from_dict("Choose a cluster policy", cluster_policies))
                 instance_profile, spark_conf_dict = self._get_ext_hms_conf_from_policy(cluster_policy)
 
-        if self._prompts.confirm("Do you want to follow a policy to create clusters?"):
-            cluster_policies_list = {f"{_.name} ({_.policy_id})": _.policy_id for _ in self._ws.cluster_policies.list()}
-            custom_cluster_policy_id = self._prompts.choice_from_dict("Choose a cluster policy", cluster_policies_list)
-        else:
-            custom_cluster_policy_id = None
+        logger.info("Creating UCX cluster policy.")
+
+        try:
+            for policy in self._ws.cluster_policies.list():
```

Comment:
for this to work, you have to add a mandatory `policy_id` field to `WorkspaceConfig`, so that it's always there.